community/ecosystem-projects.md

```diff
@@ -0,0 +1,49 @@
+# Ecosystem Projects
+
+The Argoproj consists of four core projects and set of supporting ecosystem projects. The ecosystem projects implement
+experimental features or opinionated use cases that are useful for the sub-set of users.
+
+## Eligibility
+
+The ecosystem project must enhance one of the existing core projects as opposed to focusing on a totally new area not related to Argo.
+
+**Experimental Feature**
+
+The project might be an experiment implementation of a new feature that eventually will be merged into one of the core projects. A good example
+is https://github.com/argoproj-labs/applicationset project that simplifies managing Argo CD applications. The functionality is developed in
+a separate repository to increase development velocity and simplify onboarding early adopters.
+
+Once project is mature enough it will be merged into the core project repository.
+
+
+**Opinionated Enhancement**
+
+The project might provide an implementation of an opinionated feature that is useful for a sub-set of core project users and should be distributed separately.
+The good examples are:
+
+* [argocd-operator](https://github.com/argoproj-labs/argocd-operator) - a Kubernetes operator for managing set of Argo CD.
+* [argocd-notifications](https://github.com/argoproj-labs/argocd-notifications) - integrates Argo CD with various notification services such as Slack, SMTP etc.
+* [argocd-image-updater](https://github.com/argoproj-labs/argocd-image-updater) - automates Docker registry monitoring and updates image tags in Argo CD applications when new image version get pushed.
+
```

Comment:
Let's call out how things like gitops-engine, pkg, argo-ui fit into the categorization. They are not really ecosystem projects, because they are dependencies to the core project.

Reply:
Other ones to call out are helm chart repo, and homebrew tap. These wouldn't belong to argoproj-labs.

Reply:
done: https://github.com/argoproj/argoproj/pull/17/commits/51175656fe60934f9849bbebe501d491f5b98d4f